### DISCUSSION TITLE
[Question]: should Frigate have multiple labels on an event?

### DISCUSSION THREAD

**Discussion by autumnwalker** [blakeblackshear/frigate#11719] [Ask A Question] [answered]
### What is your question:

Right now I only get one label on an event / detection / alert even though there are clearly other items visible. E.g. I will get a person label from someone in the driveway, but there is also a car visibile.

Am I missing something or is this expected behaviour?

**Comment by NickM-27** [comment]
This is expected behavior for frigate 0.13

The next version of frigate combines multiple objects into a single event

**Reply by autumnwalker** [reply]
So if I'm running 0.14.0-beta2 I should see multiple objects in a single event?

**Reply by NickM-27** [reply]
Yes, assuming they multiple objects were active at the same time

**Reply by autumnwalker** [reply]
Should it detect multiples of the same object type (i.e. person, multiple people)?

**Reply by NickM-27** [reply]
Not sure what you mean

**Reply by autumnwalker** [reply]
Running 0.14.0-beta2

I have some events in my driveway where there are multiple vehicles and multiple people.

I can understand why the vehicles are not marked as they have timed out, but there are multiple people moving in the event. Of those people, only one is tagged as a "person" object and the other people are not tagged (i.e. no box around them).

**Reply by NickM-27** [reply]
bounding boxes are not shown in 0.14 UI so it seems you may not be running 0.14

**Reply by autumnwalker** [reply]
They show in the Frigate+ panel for me.

**Reply by NickM-27** [reply]
oh right, that is working as expected

**Reply by autumnwalker** [reply]
Ok - so 0.14 will do multiple objects, but only one of each object type. So:

If there is an active car + active person both will have a bounding box.

If there are two active people only one person will have a bounding box.

**Reply by NickM-27** [reply]
you are confusing different topics. The original question was about events, which are replaced by review items in the 0.14 UI. These review items combine all objects types based on activity. 

The frigate+ review page continues to work the same as events did in 0.13

**Reply by autumnwalker** [reply]
Sorry - I guess I am; lots of terms.

This question stems from submitting things to Frigate+. I see multiple items in any given "event" that I submit, but Frigate only ever puts a bounding box on one item. So I'm left labeling repeat objects over and over again (e.g. parked vehicles).

I'm wondering if one bounding box per Frigate+ "event" is expected behaviour? I would have expected that Frigate would put a bounding box on anything in the frame for submission to Frigate+ as I am expected to do.

**Reply by NickM-27** [reply, marked as answer]
yes, this is expected. This is because frigate only saves these snapshots for the "best" moment for each object. These don't necessarily overlap for example between two people so separate images will exist. 

Automatic bounding box labeling for frigate+ will come at some point in the future

**Reply by autumnwalker** [reply]
Awesome thanks for taking the time to explain it to me Nick!
